### PR TITLE
Fix hyperlinks in Excel report

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1059)

### CODE REVIEW

File: utils/src/main/kotlin/Extensions.kt [hunk 1]
```diff
@@ -200,6 +200,17 @@ fun String.isSemanticVersion(type: Semver.SemverType = Semver.SemverType.STRICT)
...
+fun String.isValidUri() =
+        try {
+            URI(this)
+            true
+        } catch (e: URISyntaxException) {
+            false
+        }
+
 /**
  * True if the string is a valid URL, false otherwise.
```
Comment (sschuberth) [inline utils/src/main/kotlin/Extensions.kt]: Unrelated, but mind adding a new commit which also links `[URL]` here?

File: reporter/src/main/kotlin/reporters/ExcelReporter.kt [hunk 2]
```diff
@@ -24,7 +24,7 @@ import ch.frankel.slf4k.*
 import com.here.ort.model.ScanRecord
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/ExcelReporter.kt]: Commit message nits:
- "the the"
- "hyerplink"

File: utils/src/main/kotlin/Extensions.kt (hunk 1 above)
Comment (mnonnenmacher) [reply utils/src/main/kotlin/Extensions.kt]: Done.

File: reporter/src/main/kotlin/reporters/ExcelReporter.kt (hunk 2 above)
Comment (mnonnenmacher) [reply reporter/src/main/kotlin/reporters/ExcelReporter.kt]: Done.

**Pull request merged**